### PR TITLE
Clarify OS requirements

### PR DESCRIPTION
**Pull request by davidjarzebowski**
PR to clarify the requirements for a `backup-utils` host based on an internal discussion.

### CODE REVIEW

**Review by shcorbett — APPROVED** [review]
This looks good to me and removes the ambiguity of "highly recommended". I would like to get feedback from @djdefi as well to make sure it aligns with support

**Review by djdefi — APPROVED** [review]

File: docs/requirements.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ storage and must have network connectivity with the GitHub Enterprise Server app
 
 ## Backup host requirements
 
-Backup host software requirements are modest: Linux or other modern Unix operating
-system (Ubuntu is highly recommended) with [bash][1], [git][2], [OpenSSH][3] 5.6 or newer, [rsync][4] v2.6.4 or newer, and [jq][11] v1.5 or newer.
+Backup host software requirements are modest: Ubuntu Linux operating system with [bash][1], [git][2], [OpenSSH][3] 5.6 or newer, [rsync][4] v2.6.4 or newer, and [jq][11] v1.5 or newer.
```
Comment (gamefiend) [inline docs/requirements.md:8]: Nice!

Comment (eisenhowerj) [reply docs/requirements.md:8]: Does this mean using Redhat/Alma as a backup server will no longer be supported?

Comment (davidjarzebowski) [reply docs/requirements.md:8]: @eisenhowerj : Thanks for the question. I'm actually in the process of updating the requirements section to make the result of our internal discussion clear. I updated the PR to `draft` while I make that change.

**Review by djdefi — APPROVED** [review]

File: docs/requirements.md (hunk 1 above)
Comment (davidjarzebowski) [reply docs/requirements.md:8]: @eisenhowerj : I have updated this PR to make it clearer how non-Ubuntu operating systems are supported. As a result I'll mark this conversation as resolved.

**Pull request merged**